pushapkscript/script.py

```diff
@@ -31,7 +31,7 @@
         jarsigner.verify(context, apk_path)
         manifest.verify(context, apk_path)
 
-    if task.extract_android_product_from_scopes(context) == 'focus':
+    if task.extract_android_product_from_scopes(context) in ['focus', 'reference-browser']:
         log.warning('Focus does not upload strings automatically. Skipping Google Play strings search.')
```

Comment:
Nit: Sorry I should have caught this earlier. This message won't be true when executed on reference-browser.